md2conf/__init__.py

```diff
@@ -5,7 +5,7 @@
 Confluence API endpoints to upload images and content.
 """
 
-__version__ = "0.2.2"
+__version__ = "0.2.3"
```

Comment:
Please don't increment the version number because other changes might get on the same release train.